pkg/virt-operator/install-strategy/strategy.go

```diff
@@ -175,7 +175,10 @@ func dumpInstallStrategyToBytes(strategy *InstallStrategy) []byte {
 		marshalutil.MarshallObject(entry, writer)
 	}
 	for _, entry := range strategy.crds {
-		marshalutil.MarshallObject(entry, writer)
+		b, _ := yaml.Marshal(entry)
+		writer.Write([]byte("---\n"))
+		writer.Write(b)
+		// marshalutil.MarshallObject(entry, writer)
```

Comment:
this commented out code needs to be removed

Reply:
Thanks